### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Filegroups_Add_ScreenReader: Visual name of "Filegroup name" edit field is programmatically not associate with edit field.

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio#25214]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG4.1.2; #FTP; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio; #Linux; #ORCA; #Name Role Value; 

### Environment Details:
Application Name: Azure Data Studio
OS Name: Ubuntu 22.04.2 LTS
Azure data studio 1.47.0
Screen Reader: ORCA

### Repro Steps:

1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Start ORCA screen reader.
4. Tab till "Database" and press enter to expand the tree.
5. Tab till any database and pressF10 to open context menu.
6. Tab till "Properties Preview" and press enter.
7. Tab till "Filegroups" tab and press enter.
8. Tab till "Add" button and press enter.
9. Tab till "Filegroup name" edit field.
10. Observed that ORCA is not announcing the label name "Filegroup name" edit field.

### Actual Result:
ORCA is not announcing the label name "Filegroup name" for the "Filegroup name" edit field. Visual name of "Filegroup name" edit field is programmatically not associate with edit field. 

### Expected Result:
Visual name of "Filegroup name" edit filed should be programmatically associate with edit filed. ORCA should be announce the  label name "Filegroup name" for the "Filegroup name" edit field.

### User Impact:
Screen Reader users will face difficulties to understand the purpose of the edit field if label name is not announced.

### Attachment:
[25214.zip](https://github.com/microsoft/azuredatastudio/files/13949017/25214.zip)

**Comment by kisantia** [comment]
@Swati1700 this has been fixed. Please install the l[atest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch) and verify the fix.

**Comment by kisantia** [comment]
@Swati1700 this has been fixed. Please install the [latest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch) and verify the fix.

**Comment by kisantia** [comment]
@Swati1700 this has been fixed. Please install the [latest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch) and verify the fix.

**Comment by Swati1700** [comment]
Verified the issue in the above build and the issue is fixed. Hence closing the bug, please find the closing attachment.  
[25214_Closed.zip](https://github.com/microsoft/azuredatastudio/files/14208928/25214_Closed.zip)
